.jshintrc

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
.jshintrc will have all the packages we need in our library, so that it allows he syntax/keywords of that specific package hence from all I gathered, I think we should keep it in the repo. Although I am looking for a replacement.